src/protojson/JobErrorMsg.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_PROTOJSON_JOBERRORMSG_H
+#define LSST_QSERV_PROTOJSON_JOBERRORMSG_H
+
+// System headers
+#include <chrono>
+#include <map>
+#include <memory>
+#include <set>
+#include <string>
+
+// Third party headers
+#include "nlohmann/json.hpp"
+
+// qserv headers
+#include "global/clock_defs.h"
+#include "global/intTypes.h"
+#include "protojson/WorkerQueryStatusData.h"
+
+// This header declarations
+namespace lsst::qserv::protojson {
+
+/// This class handles the message used to inform the czar that there has
+/// been a problem with an UberJob.
+class JobErrorMsg {
```

Comment:
Is this class for a "job" or for an "uber-job"? According to the comment, it's the latter. So, perhaps a better name would be `UberJobErrorMsg`?